spec/game/board_spec.rb

```diff
@@ -35,6 +35,21 @@ module Game
       it "is equal to another board when all positions are the same" do
         expect(@board).to eq Board.new
       end
+
+      it "provides positions by row" do
+        expect(@board.get_rows.length).to eq 3
+      end
+    end
+
+    context "a 4x4 board" do
+      before(:each) do
+        @board = Board.new(4)
```

Comment:
let(:board) { Board.new(4) }

https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let